ONSdigital/ras-rm-docker-dev
# Run Python services using volumes. Add `local` make target

# Motivation and Context
Allows Python services to be run from a mounted Volume in docker containers. This allows hot-reloading of code-changes, creating a tighter feedback cycle for developers.


# What has changed
- Added a new `ras-local.yml` docker-compose file.
- Added the `make` target `local` that will start all RAS Python services mounted under a Volume using docker-compose.

# How to test?
- Use `make local` to start the local Python services.
- Make sure all the relevant repos are up-to-date with latest Master on Github
- Run the acceptance tests against the environment, ensuring they pass
- Make arbitrary changes to the codebase to check that hot-reloading is on

# Links
https://trello.com/c/4O9axOVO/155-mount-python-services-using-volumes-in-docker-containers-8
